### PR TITLE
CI Update - remove shims from .NET Core

### PR DESCRIPTION
**Pull request by mairaw**
@roji @carlossanlop @safern 

This will remove OdbcParameter.Offset among others that were incorrectly identified as .NET Core:
https://review.docs.microsoft.com/en-us/dotnet/api/system.data.odbc.odbcparameter.offset?view=xamarinmac-3.0&branch=remove-shims

### CODE REVIEW

File: xml/Microsoft.Extensions.Configuration/ChainedBuilderExtensions.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
         <ReturnType>Microsoft.Extensions.Configuration.IConfigurationBuilder</ReturnType>
       </ReturnValue>
       <Parameters>
-        <Parameter Name="configurationBuilder" Type="Microsoft.Extensions.Configuration.IConfigurationBuilder" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" RefType="this" />
+        <Parameter Name="configurationBuilder" Type="Microsoft.Extensions.Configuration.IConfigurationBuilder" RefType="this" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
```
Comment (mairaw) [inline xml/Microsoft.Extensions.Configuration/ChainedBuilderExtensions.xml]: @joelmartinez can we make sure this doesn't change position from one version to the next?

Comment (joelmartinez) [reply xml/Microsoft.Extensions.Configuration/ChainedBuilderExtensions.xml]: @mairaw we'll probably need to do some parameter sorting to make sure ... @mimisasouvanh, can you open a devops item on this? the functionality wouldn't be changing, just doing some sorting to maintain a consistent order

Comment (mairaw) [reply xml/Microsoft.Extensions.Configuration/ChainedBuilderExtensions.xml]: I can file one. Just point me to the right DevOps instance. I never know. That's why filing in the repo was easier 😉

Comment (joelmartinez) [reply xml/Microsoft.Extensions.Configuration/ChainedBuilderExtensions.xml]: lol ... ok ... here :) https://ceapex.visualstudio.com/Engineering/

**Comment by mairaw** [comment]
I don't think this PR introduced these. I'll have to look at those later.

**Pull request merged**